src/Microsoft.OData.Core/SelectedPropertiesNode.cs

```diff
@@ -204,6 +210,19 @@ internal enum SelectionType
             PartialSubtree = 2,
         }
 
+        /// <summary>
+        /// Gets names of the expanded entity nodes at current level.
+        /// </summary>
+        internal IEnumerable<string> NamesOfSelectedExpandedEntities
+        {
+            get
+            {
+                return this.children == null
+                    ? Enumerable.Empty<string>()
+                    : this.children.Where(_ => _.Value.isExpandedNavigationProperty).Select(c => c.Key);

```

Comment:
>_ [](start = 42, length = 1)

I prefer to use a somewhat descriptive name, such as "node" or "childNode"

Reply:
renamed.

---
In reply to: [229463175](https://github.com/OData/odata.net/pull/1106#discussion_r229463175) [](ancestors = 229463175)